addon/components/infinity-loader.js

```diff
@@ -14,8 +14,21 @@ export default Ember.Component.extend({
   developmentMode: false,
   scrollable: null,
 
+  didRender() {
+    this._super(...arguments);
+    if(Ember.VERSION >= '1.13.0') {
+      this._setup();
+    }
+  },
+
   didInsertElement() {
     this._super(...arguments);
+    if(Ember.VERSION < '1.13.0') {
```

Comment:
i'd prefer we use Ember Version Is - The idea behind it is that as we stop supporting old versions of ember, we simply go through and snip those blocks.  It's more a maintainability thing than anything else.